Helper/Config.php

```diff
@@ -1488,6 +1517,10 @@ public function getAllConfigSettings()
         $boltSettings[] = $this->boltConfigSettingFactory->create()
                                                          ->setName('title')
                                                          ->setValue($this->getTitle());
+        // Composer version
+        $boltSettings[] = $this->boltConfigSettingFactory->create()
+                                                         ->setName('composer_version')
+                                                         ->setValue($this->getComposerVersion());
```

Comment:
@vitaliyreznikov @matt-thomason   I’ve tried to set the composer version to other places, they are added to the Magento API response, but they aren’t added to the Debug Endpoint result. See here -  https://prnt.sc/tmmjsz https://prnt.sc/tmmmwh

I believe we need the Bolt internal team to add a new mapping rule so that the new property can be added to the Debug Endpoint result. Should we still go ahead and make the change here and then follow up with the Bolt internal team?

Reply:
@johnmccombs1 @vitaliyreznikov I agree with Vitaliy here, lets bubble this up like we do for phpVersion and the like, and I'll create a backlog ticket for MX to add `composerVersion` to the result in storm.

Reply:
Backlog item created: https://boltpay.atlassian.net/browse/MX-1555

Reply:
@matt-thomason @vitaliyreznikov I've set the composer version as php version. Can you re-check?

Reply:
Love it, thanks 👍